webhook/webhook_test.go

```diff
@@ -699,8 +699,9 @@ func setUserAnnotation(userC, userU string) jsonpatch.JsonPatchOperation {
 func NewAdmissionController(client kubernetes.Interface, options ControllerOptions,
 	logger *zap.SugaredLogger) (*AdmissionController, error) {
 	return &AdmissionController{
-		Client:  client,
-		Options: options,
+		Client:                client,
```

Comment:
do we actually want a test, where something will fail?

Reply:
Added one.